server/controllers/profile.js

```diff
@@ -2,8 +2,32 @@ const uploadImage = require("../middleware/uploadImage");
 const User = require("../models/User");
 const mongoose = require("mongoose");
 
-exports.postAddImage = async (req, res, next) => {
+exports.postUpdateProfile = async (req, res, next) => {
```

Comment:
If youre using promise chains on the User.findOneAndUpdate method, no need for this to be async function

Reply:
Thanks! I would change it